lib/hanami/cli.rb

```diff
@@ -131,6 +131,17 @@ def routes
       end
     end
 
+    desc 'secret', 'Prints a secure hash'
+    long_desc <<-EOS
+    `hanami secret` will generate a secure key to be used by the developer as they want, e.g. cookie sessions.
+    EOS
+    def secret(length = '64')
+      require 'securerandom'
+      # must divide by 2 because SecureRandom.hex multiplies it by 2
+      divided_length = length.to_i / 2
```

Comment:
would rather name this `number_of_bytes` rather than `divided_length`, but I also realized his solution might not be as good as I thought....

Not sure if this is likely, but if someone runs `hanami secret 65`, they'll get a string that's 64 characters long. Maybe we should keep it as it was? What do you think? 😇